gbpservice/nfp/configurator/agents/vpn.py

```diff
@@ -0,0 +1,275 @@
+import json
+import os
+import oslo_messaging as messaging
+import requests
+
+from gbpservice.nfp.configurator.agents import agent_base
+from gbpservice.nfp.configurator.lib import data_filter
+from gbpservice.nfp.configurator.lib import utils
+from gbpservice.nfp.configurator.lib import vpn_constants as const
+from gbpservice.nfp.core import main
+
+from oslo_log import log as logging
+from oslo_messaging import MessagingTimeout
+
+from neutron import context as ctxt
+
+
+LOG = logging.getLogger(__name__)
+
+
+class VpnaasRpcSender(data_filter.Filter):
+    """ RPC APIs to VPNaaS Plugin.
+    """
+    RPC_API_VERSION = '1.0'
+    target = messaging.Target(version=RPC_API_VERSION)
+
+    def __init__(self, context, sc):
+        self.context = context
+        self.sc = sc
+        self.notify = agent_base.AgentBaseNotification(sc)
+        super(VpnaasRpcSender, self).__init__(None, None)
```

Comment:
Why do we need to pass the None arguments explicitly? Filter library is not using these variables? can we get rid of them? Discuss with Pritam on why these are needed in Filter lib.